tools/install_demo_configuration.bat

```diff
@@ -62,6 +62,11 @@ if %cluster_mode% == 0 (
     )
 )
 
+if %assumeyes% == 1 (
```

Comment:
for my knowledge, what would this do?

Reply:
It sets all of the individual flags to true, `assumeyes` mean yes to all of `initsecurity`, `clustermode` and `skip_updated` (which is already defaulted to true).

Good catch @scrawfor99 !

Reply:
No is the default, if `-y` is not specified then you get prompted for input to choose. 

`-y` advertises that it accepts all prompts, but its not working as advertised without this fix.

Reply:
ah..gotcha.. thank you @cwperks!